lms/envs/common.py

```diff
@@ -1031,6 +1028,42 @@
 ################################# Jasmine ##################################
 JASMINE_TEST_DIRECTORY = PROJECT_ROOT + '/static/coffee'
 
+
+######################### Branded Footer ###################################
+# Constants for the footer used on the site and shared with other sites
+# (such as marketing and the blog) via the branding API.
+
+# URL for OpenEdX displayed in the footer
+FOOTER_OPENEDX_URL = "http://open.edx.org"
+
+# URL for the OpenEdX logo image
+# We use logo images served from files.edx.org so we can (roughly) track
+# how many OpenEdX installations are running.
+# Site operators can choose from these logo options:
+# * https://files.edx.org/openedx-logos/edx-openedx-logo-tag.png
+# * https://files.edx.org/openedx-logos/edx-openedx-logo-tag-light.png"
+# * https://files.edx.org/openedx-logos/edx-openedx-logo-tag-dark.png
+FOOTER_OPENEDX_LOGO_IMAGE = "https://files.edx.org/openedx-logos/edx-openedx-logo-tag.png"
+
+# This is just a placeholder image.
+# Site operators can customize this with their organization's image.
+FOOTER_ORGANIZATION_IMAGE = "images/default-theme/logo.png"
```

Comment:
@singingwolfboy I've moved this configuration here rather than in the `footer.html` template so the information from the API will be consistent with the templates.  I'm not sure what impact this will have on theming, so please let me know if there's a better approach.